### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2024-06-27

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#166] [General]
Previous agenda: #165 

### Agenda
- [x] Introduce any new attendees.
- [x] Summarize any PRs merged since last meeting.
- [ ] Review release follow-up issues and PRs
- [ ] Nominate work for next release
- [ ] Open next meeting discussion

**Comment by xfiderek** [comment]
1. Discuss the progress on simplifying build of downstream images. Discuss the diagram and get buy-in:
![generate_repos drawio(3)](https://github.com/space-ros/space-ros/assets/57559956/371887c8-50e9-48fc-b403-85013561af80)

_Core/dev versions of spaceros images are excluded from the diagram for the sake of simplicity_

2. Rviz and gazebo in downstream images (e.g. nav2) - my proposal is to keep rviz and gazebo separate from spaceros images. The reason is that it since we install everything from source, we would also need to install those from source. Maybe we can use standard ROS2 images for gazebo/rviz related components in demos and spawn them in separate containers.

**Reply by xfiderek** [reply]
Reg point 2. we agreed on using base ros images (e.g. humble-desktop) for rviz/gazebo nodes in order to keep spaceros images minimal

**Comment by EzraBrooks** [comment]
Go over the shenanigans Erik and I caused

**Comment by Bckempa** [comment]
Next agenda: https://github.com/space-ros/space-ros/discussions/168
